src/store/link.ts

```diff
@@ -0,0 +1,38 @@
+import { createSlice, PayloadAction, createSelector } from "@reduxjs/toolkit";
+import { getParticipants } from "./zoom";
+
+interface LinkedAccountsState {
+  byId: {
```

Comment:
See my comment below, but I'm unconvinced that this object (and by extension, this slice) should exist separately from the `zoom` slice.

If you do get rid of this slice, please keep the other one nicely set up as a standalone slice, like it is now! That way, we can add more slices in the future if we need.